### PR TITLE
openmodelica newsletter december-2020

### PR DESCRIPTION
**Pull request by arun3688**

### CODE REVIEW

File: _2020-12/ModelicaConference2021.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+More information is contained on the conference page https://2021.international.conference.modelica.org
+
+You are encouraged to submit a full paper until April 26, 2021. Every submitted paper will be reviewed by 3 reviewers. 
+More details about the "call for papers", "call for libraries", "call for vendor presentations", "call for exhibitors", "call for sponsors"
+and "call for tutorials/workshops" is available under: https://2021.international.conference.modelica.org/call2021.html 
+
+We are looking forward to seeing you in Linköping.
+
+Lena Buffoni, Lennart Ochel, Adrian Pop and Martin Sjölund \
+Organizers
```
Comment (sjoelund) [inline _2020-12/ModelicaConference2021.md:20]: We already have #162

Comment (arun3688) [reply _2020-12/ModelicaConference2021.md:20]: @sjoelund, i think peter did not know about the other #162, Should i remove this article from this PR ?

Comment (sjoelund) [reply _2020-12/ModelicaConference2021.md:20]: Yes

Comment (arun3688) [reply _2020-12/ModelicaConference2021.md]: @sjoelund removed

**Pull request merged**